test/e2e/tests/single_zone_e2e_test.go

```diff
@@ -199,7 +199,7 @@ var _ = Describe("GCE PD CSI Driver", func() {
 		Expect(cloudDisk.SizeGb).To(Equal(newSizeGb))
 
 		// Resize node
-		err = client.NodeExpandVolume(volID, publishDir, newSizeGb)
+		_, err = client.NodeExpandVolume(volID, publishDir, newSizeGb)
```

Comment:
We may also want an "offline" test case where NodeExpand is called after NodeStage, but before NodePublish

Reply:
done